spring-integration-core/src/main/java/org/springframework/integration/support/locks/RenewableLockRegistry.java

```diff
@@ -16,12 +16,15 @@
 
 package org.springframework.integration.support.locks;
 
+import org.springframework.scheduling.TaskScheduler;
+
 /**
  * A {@link LockRegistry} implementing this interface supports the renewal
  * of the time to live of a lock.
  *
  * @author Alexandre Strubel
  * @author Artem Bilan
+ * @author NaccOll
```

Comment:
Forgive me for my official tone, but this does not sound like a legal name to be applied for the license.